content/en/security/application_security/threats/library_configuration.md

```diff
@@ -33,7 +33,11 @@ ASM automatically attempts to resolve `http.client_ip` from several well-known h
 
 ## Track authenticated bad actors
 
-Many critical attacks are performed by authenticated users who can access your most sensitive endpoints. To identify bad actors that are generating suspicious security activity, add user information to traces by instrumenting your services with the standardized user tags. You can add custom tags to your root span, or use instrumentation functions. Read [Tracking User Activity][1] for more information.
+Many critical attacks are performed by authenticated users who can access your most sensitive endpoints. To identify bad actors that are generating suspicious security activity, add user information to traces by instrumenting your services with the standardized user tags. You can add custom tags to your root span, or use instrumentation functions.
+
+The Datadog Tracing Library will attempt to auto-detect user login and signup events when it detects compatible authentication frameworks in use.
```

Comment:
The Datadog Tracing Library attempts to detect user login and signup events when it detects compatible authentication frameworks in use.

Remove `will` and `auto`; the auto part of `auto-detect` might be a little redundant, as in the context of the tracing library I think it implies that the library will do it.

Reply:
Should we also mention that ASM needs to be active for it to work?

Reply:
Updating to:

> The Datadog Tracing Library attempts to detect user login and signup events when compatible authentication frameworks are in use, and ASM is enabled.